src/App.jsx

```diff
@@ -14,70 +15,97 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const defaultSelectedGood = 'Jam';
+const emptySelectedGood = '';
+const noGoodsSelectedMessage = 'No goods selected';
```

Comment:
I don't think it needs to be put into a separate variable, but I'm not sure
(emptySelectedGood, noGoodsSelectedMessage)